skyportal/handlers/api/photometry.py

```diff
@@ -138,6 +140,24 @@ def post(self):
 
         if "altdata" in data and not data["altdata"]:
             del data["altdata"]
+
+        # quick validation - just to make sure things have the right fields
+        try:
```

Comment:
what happens if someone tries to post both fluxes and mags (by mistake)?

Reply:
I guess the fluxes are just ignored then

Reply:
It will fail with "unknown column"

Reply:
In both cases (unknown column Mag if attempted to parse as flux, unknown column flux if attempted to parse as mag)